Makefile.am

```diff
@@ -438,6 +441,24 @@ myhtopplatsources = $(unsupported_platform_sources)
 myhtopplatheaders = $(unsupported_platform_headers)
 endif
 
+# Sorting the signal list at build time
+# -------------------------------------
+
+SUFFIXES = .i
+# Preprocessing a list of signals to get their numbers from the headers.
+.c.i:
```

Comment:
```suggestion
signals/SignalList.in.i: signals/SignalList.in.c
```

Reply:
In the suggested form, this won't work with out-of-tree builds. Why not use the suffix rule? I believe that suffix rules are idiomatic for Automake and the recipe is generic.

Reply:
The suffix rule would affect all files with the same extension in the build tree. Since we have just one file that needs such preprocessing, a suffix rule would be a bad idea.

And yes, the dependency line should be ```$(srcdir)/signals/SignalList.in.c``` instead.